R/mechanism-objective.R

```diff
@@ -19,7 +19,7 @@ mechanismObjective$methods(
         x <- x[, cols]
 
         # censor & impute missing values
-        x <- censorData(x, .self$varType, .self$rng, .self$bins)
+        x <- censorData(x, .self$varType, .self$rng, .self$bins) # NEEDS fixed to support checkRange
```

Comment:
commenting here just to flag the existing comment about fixing this to support checkRange()